### PR TITLE
[cloud-provider-*] Change service account from default to csi for csi-node drivers

### PR DESCRIPTION
**Pull request by vitaliy-sn**
## Description

Changed the service account from `default` to `csi` for csi-node drivers

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-*
type: fix
summary: Changed the service account from default to csi for csi-node drivers
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/azure'...

**Comment by vitaliy-sn** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 9255894703 9255894703-1-con-1-25 /sys/deckhouse-oss/install:pr8498 redos@158.160.40.144

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: ee/modules/030-cloud-provider-openstack/templates/csi/controller.yaml [hunk 1]
```diff
@@ -69,6 +69,7 @@
   {{- $csiNodeConfig := dict }}
   {{- $_ := set $csiNodeConfig "nodeImage" $csiControllerImage }}
   {{- $_ := set $csiNodeConfig "driverFQDN" "cinder.csi.openstack.org" }}
+  {{- $_ := set $csiNodeConfig "serviceAccount" "csi" }}
```
Comment (asviel) [inline ee/modules/030-cloud-provider-openstack/templates/csi/controller.yaml:72]: Maybe we should change default value in `helm_lib_csi_node_manifests` template?

Comment (vitaliy-sn) [reply ee/modules/030-cloud-provider-openstack/templates/csi/controller.yaml]: Good point. I'll fix the template.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/gcp'...

**Comment by vitaliy-sn** [comment]
/e2e/abort azure;Standard;containerd;1.25 9497558836 9497558836-1-con-1-25 /sys/deckhouse-oss/install:pr8498 azureuser@172.211.131.87

**Comment by vitaliy-sn** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 9497559152 9497559152-1-con-1-25 /sys/deckhouse-oss/install:pr8498 centos@54.93.199.34

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by vitaliy-sn** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 9498446038 9498446038-1-con-1-25 /sys/deckhouse-oss/install:pr8498 centos@35.159.186.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've recognized your '/e2e/abort' command and started the workflow...

**Pull request closed without merging**